src/logic/getBrowserStoreData.ts

```diff
@@ -1,5 +1,5 @@
-export default (storageType: Storage, storeName: string) => {
-  const sessionStorageData = storageType.getItem(storeName);
+export default async (storageType: Storage, storeName: string) => {
+  const sessionStorageData = await storageType.getItem(storeName);
   try {
```

Comment:
Maybe we should move the await also in the try catch block.

Reply:
sounds good to me.